.circleci/config.yml

```diff
@@ -64,6 +67,11 @@ defaults:
       path: build/solc/solc
       destination: solc
 
+  # windows binary archive
+  - artifact_solc_windows: &artifact_solc_windows
+      path: upload/solidity-windows.zip
```

Comment:
Does it actually still need to be a zip?

Reply:
Well, currently it builds all binaries, so the compiler, the phaser and the upgrade tool...
If we only want to distribute ``solc.exe``, then it doesn't need to be a zip file anymore :-). But it'd probably also break the new binary distribution scheme of @cameel's, wouldn't it :-)?

Weirdly enough the zip files of our previous releases did not include the phaser or the upgrade tool, but only ``solc.exe``, the required DLLs and then ``soltest.exe`` for whatever reason...

Reply:
Hm... well... what to release generally is a different question from what to put up as artifacts here, though.
I could change it to upload the three binaries separately as artifacts? I don't mind either way.

Reply:
I don't think replacing the zip with an exe would be a big problem as long as it's still a single file. We have a file list so external tools will be able to find it. They just need to be adjusted to handle both possibilities and not try to unpack the exe so it would be nice to give some advance notice to tool authors.

Reply:
The reason for the zip file was the requirement to ship the dlls. If that is not an issue anymore, I think we should ship the .exe.

Reply:
Makes sense. And we can still decide and think about if and where and how we want to distribute the phaser and upgrade tool.
I'll change this PR to upload all three executables separately as artifacts in any case.

Reply:
When is the next release? If it's this week, then I'd keep it as a zip until then, not to introduce this change too suddenly... We'll need some minor changes in the `update` script for it to pick up the exe.

Reply:
we still have the zip from appveyor. I would not cut the appveyor run until after the next release.